incubator/hnc/test/e2e/issues_test.go

```diff
@@ -82,7 +82,9 @@ var _ = Describe("Issues", func() {
 		MustRun("kubectl delete subns", nsChild, "-n", nsParent)
 	})
 
-	It("Should remove obsolete conditions CannotPropagateObject and CannotUpdateObject - issue #328", func() {
+	// This issue is obsolete after introducing events to surface errors in the
+	// object reconciliation, thus the e2e test case is obsolete too.
+	PIt("Should remove obsolete conditions CannotPropagateObject and CannotUpdateObject - issue #328", func() {
```

Comment:
Any reason not to just delete these?

Reply:
Deleted and have the issue IDs in the new `Should` clause saying `replacing obsolete issues x, y, z`.

Reply:
If the issues are truly obsolete IMO we don't need a record of them :)